README.md

```diff
@@ -5,7 +5,19 @@ Faro
 For a quick start follow the instructions below. For more in depth information on why and how we build Faro, the [wiki](https://github.com/icapps/ios-faro/wiki) page.
 
 ## Concept
-We build a service request by using a `Bar` class as the point where you fire your `Call` and get a `Result`.
+We build a service request by using a `Service` class as the point where you fire your `Call` and get a `Result`.
+
+### features
+
+*Service*
+* Service written to use Swift without Objective-C
+* Service cleanly incapsulates all the parameters to handle a netowerk request
+* Easily write a 'MockService' to load JSON from a local drive
+
+*Automagically Parse*
+* Automatic Serialization and Mapping thanks to the use off the Swift 'Mirror' class.
+* Uses Protocol extensions to minimize the work needed on your end 😎
```

Comment:
Ik heb call toegevoegd. Maar weet niet goed wat ik daar over moet zeggen. Is nogal straight forward maar dat kan aan mij liggen.